pkg/s3/error.go

```diff
@@ -0,0 +1,9 @@
+package s3
+
+import "github.com/bacalhau-project/bacalhau/pkg/models"
+
+const S3_PUBLISHER_COMPONENT = "S3PUB"
```

Comment:
nit: tweak to name and variable pattern, please remove underscores at a minimum.
```suggestion
const ComponentName = "S3_PUB"
```
suggesting to drop "S3" from the name since it's implied by the package (s3) containing the variable.